packages/design-system/src/Modal/Modal.styles.tsx

```diff
@@ -49,6 +49,7 @@ export const StyledContent = styled(Content)`
   display: flex;
   flex-direction: column;
   z-index: 1001;
+  max-width: 640px;
```

Comment:
max-width already exist in line 45.

Reply:
Also, will this affect any existing modals? like generate page from template and all.
Please use relative value

Reply:
@ankitakinger : We cannot define max-width coz we have modal which are full screen modal

Reply:
then shall I add width instead? For full screen modals in this case then we may have to add the inline style to define its width.